### PR TITLE
Ensure CSV file is named correctly during extract/copy

### PR DESCRIPTION
**Pull request by mikenairn**
* Check csv file name after copy and rename if it does not meet the expected pattern.
* CRW has a file named differently, and it fails to start the configmap registry that the intly operator uses.

```
./delorean ews extract-manifests --src-image registry-proxy.engineering.redhat.com/rh-osbs/codeready-workspaces-operator-metadata:2.1-47 --dest-dir /home/mnairn/go/src/github.com/integr8ly/integreatly-operator/manifests/integreatly-codeready-workspaces
```

CSV file should be named \<product>.\<version>.clusterserviceversion.yaml i.e. "crwoperator.v2.1.1.clusterserviceversion.yaml"

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
👀

File: cmd/extractManifests.go [hunk 1]
```diff
@@ -55,6 +55,15 @@ func copyLatestManifest(srcPkgDir, destPkgDir string) (string, string, error) {
...
 	}
 
+	csvFileName := filepath.Base(csvFile)
+	expectedCSVFileName := utils.GetCSVFileName(csv)
+	if expectedCSVFileName != csvFileName {
+		err := os.Rename(filepath.Join(destBundleDir, csvFileName), filepath.Join(destBundleDir, expectedCSVFileName))
+		if err != nil {
+			return "", "", err
+		}
+	}
```
Comment (wei-lee) [inline cmd/extractManifests.go:65]: it would be good if we can add a test case for it, just to ensure this will always work

**Comment by wei-lee** [comment]
/lgtm

I am ok to get this merged for now. But would prefer a test case to be added later for this use case.

**Comment by laurafitzgerald** [comment]
Hi @mikenairn. I'm going through the steps of verifying this one (running the command on this branch)
I'm getting this error `Error: no ClusterServiceVersion object found in /var/folders/dx/bnyqt7qn03s23vdg4vvfbl3h0000gn/T/manifests-340456545/v2.1.1` Is that what's expected?

File: cmd/extractManifests.go (hunk 1 above)
Comment (mikenairn) [reply cmd/extractManifests.go:65]: @wei-lee Added additional test, can you review again please.

**Comment by laurafitzgerald** [comment]
> Hi @mikenairn. I'm going through the steps of verifying this one (running the command on this branch)
> I'm getting this error `Error: no ClusterServiceVersion object found in /var/folders/dx/bnyqt7qn03s23vdg4vvfbl3h0000gn/T/manifests-340456545/v2.1.1` Is that what's expected?

You can ignore this. I wasn't building first. Woops. \lgtm

**Pull request merged**